.github/workflows/support-tools.yml

```diff
@@ -0,0 +1,41 @@
+name: Build support tools binaries
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: setup golang
+        uses: actions/setup-go@v2
+        id: go
+        with:
+          go-version: 1.22.4
+
+  build:
+    needs: [setup]
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: build tools' binaries
+        run: make build-tools-all
+
+      - name: release the binaries
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: bin/*
+          tag: support-tools
```

Comment:
did we check that this captures only the support tools and not unnecessary binaries?

Reply:
thanks for the comment, from what I [tested](https://github.com/shajmakh/numaresources-operator/releases/tag/support-tools) it should release only binaries that were built using the imported make target. It would be interesting to make sure though. I agree we should not assume that though, so I updated that to point exactly to the 2 helper tools that can be released.

Reply:
explicitely allowlisting seems to be the best option, thanks for updating